protocol/protocol.md

```diff
@@ -69,105 +77,230 @@ Syntax:
+                    "edition": "", 
+                    "major": 8, 
+                    "micro": 7, 
+                    "minor": 0, 
+                    "string": "8.0.7"
+                }
+            }, 
+            "meta": {
+                "message": null, 
+                "status": "ok", 
+                "statuscode": 100
+            }
+        }
+    }
+
+Explanation of the capabilities:
+
+* `pollinterval`: Interval to poll for server side changes (unused)
+* `bigfilechunking`: Flag if server supports big file chunking
+* `undelete`: Flag if server supports big file trash
+* `versioning`: Flag if server supports big file versioning
+* `version`: detailed server version information
+* `meta`: ocs meta information
 
 ## WEBDAV
 
-Standard webdav request/response conventions apply. 
+Standard webdav request/response conventions apply.
 
 Namespace for non-standard properties: `http://owncloud.org/ns`
 
 ### Restrictions and limitations
 
+#### ETag Format
+
+The HTTP header for the ETag can be implemented in two different formats due to historical
+error conditions of the ownCloud server. 
+
+The normal and expected format is that the value of the header contains the ETag. However, 
+if the header value is enclosed in two quote signs, the client is expected to remove the
+quotes. 
+
+Also, if the header value contains the word "-gzip", client is expected to remove that with no replace. Ref http://github.com/owncloud/client/issues/1195
+
 #### Properties
 
-The etag value must be "quoted". 
+In the response body the value of resourcetype property MUST NOT contain whitespaces (implementation as of owncloud client 1.5):
 
-In the response body the value of resourcetype property MUST NOT contain whitespaces (implementation as of owncloud client 1.5):   
-   
         <d:resourcetype><d:collection/></d:resourcetype>
 
 #### Headers
 
 PROPFIND Depth:infinity is not supported: client will try this for compatibility with older code  but server may choose to return 501 (not implemented).
 
+### Quota Check Call
 
-
-### Quota check
+Client displays user quota information. For that it does a PROPFIND call to the root of the 
+user space on the webdav server.
 
 Syntax:
-	    
-	> PROPFIND /remote.php/webdav/ HTTP/1.1
+
+	PROPFIND /remote.php/webdav/ HTTP/1.1
 	Depth: 0
-	
+
 	<?xml version="1.0" ?>
 	  <d:propfind xmlns:d="DAV:"><d:prop>
-	      <d:quota-available-bytes/>   
-	      <d:quota-used-bytes/>  
+	      <d:quota-available-bytes/>
+	      <d:quota-used-bytes/>
 	  </d:prop></d:propfind>
 
 Response: 207
 
 Reponse body example:
 
-    < PROPFIND
-    
-    <?xml version="1.0" encoding="utf-8"?>
-     <d:multistatus xmlns:d="DAV:" xmlns:s="http://sabredav.org/ns">
+    <?xml version='1.0' encoding='utf-8'?>
+    <d:multistatus xmlns:d="DAV:" xmlns:s="http://sabredav.org/ns" xmlns:oc="http://owncloud.org/ns">
       <d:response>
-        <d:href>/remote.php/webdav/</d:href>
+        <d:href>/oc/remote.php/webdav/</d:href>
```

Comment:
I will remove '/oc' from this line because it is not needed. Please correct me if I am wrong.